src/abaqus/UtilityAndView/SymbolicConstant.py

```diff
@@ -2579,58 +2644,3 @@ def __init__(self, text: str) -> None:
     MINIMUM_ENVELOPE = "MINIMUM_ENVELOPE"
     RANGE_ALL = "RANGE_ALL"
     SS = "SS"
-    
-    @abaqus_method_doc
-    def __copy__(self) -> SymbolicConstant:
-        ...
-    
-    def __getstate__(self) -> str:
-        return self.text
-    
-    # def __hash__(self) -> int:
-    #     ...
-    
-    def __lt__(self, other: Any) -> bool:
-        """Sorting method; True if self is < other"""
-        return self < other
-    
-    @abaqus_method_doc
-    def __reduce__(self) -> tuple:  # known return case of __reduce__
-        ...
-    
-    def __repr__(self) -> str: 
-        return self.getText()
-    
-    def __setstate__(self, *args) -> None:
-        ...
-    
-    def __str__(self) -> str:
-        return self.text
-    
-    def getId(self) -> int:
-        ...
-    
-    def getText(self) -> str:
-        return self.text
-    
-    # @staticmethod 
-    # def __new__(cls, name: str, *args, **kargs) -> Union[SymbolicConstant,AbaqusBoolean]:
-    #     if name == "ON":
-    #         return AbaqusBoolean(1)
-    #     elif name == "OFF":
-    #         return AbaqusBoolean(0)
-    #     else:
-    #         return super().__new__(cls)
-    
-    @classmethod
-    def _addToCache(cls, *args, **kwargs):  # real signature unknown
-        """
-        Helper method for __new__
-        Also used by AbaqusBoolean.__new__
-        """
-        pass
-
-    __weakref__ = property(lambda self: object(), lambda self, v: None, lambda self: None)  # default
-    """list of weak references to the object (if defined)"""
-
-SymbolicConstant = abaqus_class_doc(abaqusConstants)
```

Comment:
I think `abaqusConstants` class should inherit from `SymbolicConstant`  or be a alias, for type annotation be consistent.

Here is an example of warning that is raising when used in type annotation without `Literal` (which may happen in some cases)

![image](https://user-images.githubusercontent.com/57204023/193283430-44cdbed7-0ba8-4e4e-b8a0-09cb3463c356.png)

![image](https://user-images.githubusercontent.com/57204023/193283649-be689986-8745-486c-b133-f76fb9c14f7c.png)

Reply:
Right!